src/hipscat/inspection/almanac.py

```diff
@@ -80,10 +80,19 @@ def _add_files_to_namespace(self, directory, namespace=""):
                 files.append(input_path)
                 continue
 
-            input_paths = glob.glob(f"{input_path}/**.yml")
+            path_contents = file_pointer.get_directory_contents(
+                input_path,
+                append_paths=False,
+                storage_options=self.storage_options
+            )
+            path_contents = [
```

Comment:
So this is the only time that `get_directory_contesnts()` is called (outside of tests). And if I don't do the `append_paths` validation, it will append the pathway to to the file twice. 

Prior to this, the functionality of `get_directory_contents` utilized `os.listdir` which only returned the filenames that are within the specified pathway. With fsspec, it returns the filenames with the pathways already appended. 

So what really needs to happen is remove the `append_paths_to_pointer` call altogether since fsspec already does that.

Reply:
I've just pushed a change the cleans this up.